src/main/java/mate/academy/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,193 @@
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createQuery = "INSERT INTO cars (model, manufacturer_id) VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statementForCar =
+                        connection.prepareStatement(createQuery, Statement.RETURN_GENERATED_KEYS)) {
+            statementForCar.setString(1, car.getModel());
+            statementForCar.setLong(2, car.getManufacturer().getId());
+            statementForCar.executeUpdate();
+            ResultSet resultSet = statementForCar.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create car: " + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getById = "SELECT * FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.id = ? AND c.is_deleted = FALSE;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statementCar = connection.prepareStatement(getById)) {
+            statementCar.setLong(1, id);
+            ResultSet resultSet = statementCar.executeQuery();
+            if (resultSet.next()) {
+                car = getCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car by id: " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getListDrivers(car.getId()));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAllCar = "SELECT * FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getAllCar)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get list of car", e);
+        }
+        for (Car car : cars) {
+            car.setDrivers(getListDrivers(car.getId()));
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateCars = "UPDATE cars SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? and is_deleted = FALSE;";
+        String deleteCarsDrivers = "DELETE FROM cars_drivers WHERE car_id = ?;";
```

Comment:
let's move the execution of this query into a private method with its own connection